app/gui/suggestion-database/src/entry.rs

```diff
@@ -972,18 +972,16 @@ where
             TagValueResolution::Resolved(entry, chain) => {
                 let label = chain_to_label(&chain);
                 let qualified_name = entry.qualified_name();
-                let parent_module = qualified_name.parent();
-                let required_import = parent_module.as_ref().map(|n| n.to_string());
-
-                let expression = if let Some(parent) = parent_module {
-                    let in_module_name = qualified_name.name();
-                    let parent_name = parent.name();
-                    [parent_name, in_module_name].join(opr::predefined::ACCESS)
+                let required_import = Some(qualified_name.to_string());
```

Comment:
Rather than doing some conditional checks, the require import is always `qualified_name`. One would assume this adds more imports, not less. Why this fix works?

Reply:
Some expressions do not need imports in dropdowns, like simple numbers. (We also have two branches below where `required_imports` is `None`) Here, we only work with suggestion database entries, and all of them require some imports. We don't add too much because later in the execution flow, we deduplicate required imports using the same functionality as CB uses. (see `controller::graph::Handle::add_required_imports`).

The previous code used a straightforward strategy, "let's unconditionally import parent module every time" (although it also had deduplication), but here we're more sophisticated.